suite-native/navigation/src/components/Screen.tsx

```diff
@@ -24,9 +24,11 @@ type ScreenProps = {
 
 const screenContainerStyle = prepareNativeStyle<{
     backgroundColor: Color;
-}>((utils, { backgroundColor }) => ({
+    paddingTop: number;
+}>((utils, { backgroundColor, paddingTop }) => ({
     flex: 1,
     backgroundColor: utils.colors[backgroundColor],
+    paddingTop,
```

Comment:
If you added it here, you should remove paddingTop from headerStyle, also we should probably add paddingBottom

Reply:
https://github.com/trezor/trezor-suite/pull/7753/commits/122aab36e65d7592983793db5ea8f0156040be39